src/ddmd/dtemplate.d

```diff
@@ -2918,6 +2919,27 @@ private ubyte deduceWildHelper(Type t, Type* at, Type tparam)
     }
 }
 
+private Type rawTypeMerge(Type t1, Type t2)
```

Comment:
Documentation would be more than welcome

Reply:
Done